scalecube-organization/src/main/java/io/scalecube/tokens/store/VaultPathBuilder.java

```diff
@@ -0,0 +1,41 @@
+package io.scalecube.tokens.store;
+
+import com.bettercloud.vault.EnvironmentLoader;
+import io.scalecube.config.AppConfiguration;
+import java.util.Objects;
+
+final class VaultPathBuilder {
+  private String vaultTokenKeysPath;
+
+  VaultPathBuilder() {
+    String vaultSecretPathPrefix = new EnvironmentLoader()
+        .loadVariable("VAULT_SECRETS_PATH");
+    String pattern = getVaultPathPattern();
+    vaultTokenKeysPath = String.format(pattern, vaultSecretPathPrefix);
+  }
+
+  String getPath(String alias) {
+    return getVaultSecretPath().concat(alias);
+  }
+
+  private String getVaultSecretPath() {
+    if (vaultTokenKeysPath == null) {
+      vaultTokenKeysPath = String.format(getVaultPathPattern(), getVaultKeyValueEngine());

```

Comment:
I'm not sure that we should reread system and file properties every time.
Maybe will be better use it like in this PR (https://github.com/scalecube/scalecube-organization-service/pull/19/files#diff-6502b431d8e0018fe0307cc2b06acb6a)